app/controllers/upload_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UploadController < ApplicationController
```

Comment:
You shouldn't be needing this? Upload functionality should be in the `track` controller methods.